packages/oas-to-snippet/__tests__/index.test.js

```diff
@@ -27,28 +29,7 @@ const operation = {
 const formData = { path: { id: 123 } };
 
 test('should be able to accept a har override', () => {
-  const harOverride = {
-    log: {
-      entries: [
-        {
-          request: {
-            method: 'GET',
-            url: 'https://dash.readme.io/api/v1/categories/',
-            httpVersion: 'HTTPS/1.1',
-            headers: [
-              {
-                name: 'authorization',
-                value: 'Basic xxx',
-              },
-            ],
-            queryString: [],
-          },
-        },
-      ],
-    },
-  };
```

Comment:
Removed this hardcoded HAR in favor of using one that I copied from HTTPSnippet into https://github.com/readmeio/har-examples (we use this repo for testing `api` snippets).